android/iden3coreapi/src/main/java/com/iden3/iden3coreapi/Iden3Ticket.kt

```diff
@@ -0,0 +1,23 @@
+package com.iden3.iden3coreapi
+
+import iden3mobile.*
+
+class Iden3Ticket(private var ticket: Ticket) {
+
+    fun getId() : String {
+        return ticket.id
+    }
+
+    fun getLastChecked() : Long {
```

Comment:
this long represents unix time. Is `long` the usual way to represent time in Android?

Reply:
Yes, android is using long for represeting time in miliseconds